keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/MemoryCloudDriverCache.kt

```diff
@@ -105,15 +125,17 @@ class MemoryCloudDriverCache(
     }
 
   override fun availabilityZonesBy(account: String, vpcId: String, purpose: String, region: String): Set<String> =
```

Comment:
All of the other functions call the `getOrNotFound` helper method except this one.